partnerapp/src/InteractionPreviewView.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from "react";
+import { observer } from "mobx-react-lite";
+import { useParams, useHistory } from "react-router-dom";
+import { CampaignStore } from "./campaign-store";
+import { Button } from "reactstrap";
+
+export const InteractionPreviewView: FC<{
+  store: CampaignStore;
+}> = observer(({ store }) => {
+  const history = useHistory();
+  const params: { campaignId: string } = useParams();
+  const campaign = store.getCampaign(params.campaignId);
+
+  if (!campaign) {
+    console.log("Could not find campaign!", params.campaignId);
+    history.push("/campaigns");
+  }
+
+  const onBackClick = () => {
+    history.push("/campaigns");
+  };
+
+  const frameDoc = `
+  <html>
+    <head>
+    </head>
+    <body>
+    <script src="https://${
+      campaign!.interactionId
```

Comment:
I think since you have an if statement before you can now remove this bang!
```suggestion
      campaign.interactionId
```